### PR TITLE
ENG-9350 fixed multi-node cluster latency logger issues

### PR DESCRIPTION
**Pull request by dweiss88**

### CODE REVIEW

File: src/frontend/org/voltdb/utils/LatencyLogger.java [hunk 1]
```diff
@@ -125,11 +126,18 @@ public double getValueAtPercentile (double percentile) {
...
+        final String hostname;
         if (args.length != 3) {
-            System.out.println("Usage server port reportIntervalSeconds");
-            return;
-        }
+            if (args.length == 4)
+                hostname = args[3];
+            else {
+                System.out.println("Usage server port reportIntervalSeconds [hostname]");
+                return;
```
Comment (manjujames) [inline src/frontend/org/voltdb/utils/LatencyLogger.java]: What is the difference between server and hostname here?

File: src/frontend/org/voltdb/utils/LatencyLogger.java [hunk 2]
```diff
@@ -137,7 +145,7 @@ public static void main(String[] args) throws Exception {
                 throw new NumberFormatException();
         } catch (NumberFormatException e) {
             System.out.println("reportIntervalSeconds should be greater than or equal to 1");
-            System.out.println("Usage server port reportIntervalSeconds");
+            System.out.println("Usage server port reportIntervalSeconds [hostname]");
```
Comment (manjujames) [inline src/frontend/org/voltdb/utils/LatencyLogger.java]: If you make a method called usage() that prints out the usage, you don't have to change this in multiple places when you change usage.

File: src/frontend/org/voltdb/dtxn/LatencyStats.java [hunk 3]
```diff
@@ -47,25 +47,26 @@
...
         }
 
         @Override
         public Object next() {
-            return null;
+            if (returnRow) {
+                returnRow = false;
+                return new Object();
+            } else {
+                return null;
```
Comment (manjujames) [inline src/frontend/org/voltdb/dtxn/LatencyStats.java]: I don't understand what this change is.. I'll talk to you on Monday.

File: src/frontend/org/voltdb/utils/LatencyLogger.java (hunk 1 above)
Comment (dweiss88) [inline src/frontend/org/voltdb/utils/LatencyLogger.java]: server can be designated with an ip address while hostname is a specific name that the cluster recognizes as the identifier for that machine.

**Comment by manjujames** [comment]
Looks good to me now. Feel free to merge when build is blue. Use squash-merge so that the commit on LatencyStats can be avoided.

**Pull request closed without merging**